_docs/latest/use/cli/notifier.md

```diff
@@ -25,9 +25,9 @@ Following are the CLI commands you can use with Notifier:
   forseti notifier run
 
   # Send the violations by inventory index id.
-  forseti notifier run --inventory_index_id <inventory index id>
+  forseti notifier run --inventory_index_id <INVENTORY_INDEX_ID>
```

Comment:
I think you only need to edit the _docs/_latest files. Changing the _docs/latest is not needed.